ulib/FStar.Real.fsti

```diff
@@ -22,86 +22,31 @@ module FStar.Real
 
   All these operations are mapped to the corresponding primitives
   in Z3's theory of real arithmetic.
+
+  This is only a logical model of the reals. There is no extraction
+  for them, as they are an erasable type. Any operation that can observe
+  a real (comparisons, to_string, etc) must be Ghost or a proposition.
 *)
 
-val real : eqtype
+[@@erasable]
+val real : Type0
 
 val of_int : int -> Tot real
 
-(**
-  Used to extract real constants; this function is
-  uninterpreted logically. i.e., 1.1R is extracted to
-  [of_string "1.1"]
-  *)
-val of_string: string -> Tot real
+val to_string: real -> GTot string
```

Comment:
Does it make sense to keep the ghost `to_string` function?  Is there any way to execute it?

Reply:
Ah good point. There's no implementation nor axioms, so probably best to just drop it.